va/va.h

```diff
@@ -213,6 +213,8 @@ typedef int VAStatus;	/** Return status type from functions */
 #define VA_STATUS_ERROR_UNSUPPORTED_MEMORY_TYPE 0x00000024
 /** \brief Indicate allocated buffer size is not enough for input or output. */
 #define VA_STATUS_ERROR_NOT_ENOUGH_BUFFER       0x00000025
+/** \brief Indicate a synchronous operation is still running */
+#define VA_STATUS_ERROR_SYNCHRONIZATION_TIMED_OUT 0x00000026
```

Comment:
`VA_STATUS_ERROR_SYNCHRONIZATION_TIMED_OUT` I think that's too big name and SYNCHRONIZATION would be granted from function name. I would name new status just in generic manner like `VA_STATUS_ERROR_TIMEDOUT`.

Reply:
Thanks. Addressed.